ui.frontend/clientlib.config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
       ...libsBaseConfig,
       name: 'clientlib-base',
       categories: ['saas-aem-module.base'],
-      dependencies: ['saas-aem-module.dependencies'],
+      dependencies: ['saas-aem-module.dependencies', 'coralui3'],
```

Comment:
we should only load coralui3 in edit mode

Reply:
hmm, how can I do that when we do not have a dedicated page component?

Reply:
good point. 
just add it directly in the htl as a simple workaround:
`<sly data-sly-test=${wcmmode.edit && model.showAlert}`>
<sly data-sly-call="${clientlib @ categories='coralui3}"/>
...
`